docs/wiki/End-user-Guide/installation/install-client.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Client
+---
+
+# Client
+
+## How to Install Cleanroom Loader on the Client
+
+### About Launchers
+
+The recommended way to run Cleanroom Loader is with a **MultiMC-based launcher**, such as **Prism Launcher** or **PolyMC**.  
+These are officially supported because other launchers may not properly handle the removal of certain vanilla libraries.
+
+---
+
+### Manual Install
+
+> **Note:** Manual installation of Cleanroom Loader is only supported on MultiMC-based launchers.
+
+Steps:
+1. Add [Fugue](https://www.curseforge.com/minecraft/mc-mods/fugue) to your modlist  
+   *(it will not load in a Forge environment)*
+2. Add [Scalar Legacy](https://www.curseforge.com/minecraft/mc-mods/scalar-legacy) to your modlist  
+   *(it will not load in a Forge environment)*
+3. ⚠️ **Warning:** Make sure your instance does not include problematic or deprecated mods.  
```

Comment:
while using the symbol directly works, you can also use common emoji syntax - `:warning:` - which might be better.